<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_BR</class>
 <widget class="QDialog" name="Dlg_BR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{border-radius:4px;background:#409eff;color:white;font-size:12px;}
QWidget#bg{background:white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>密     码</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>用户昵称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_user"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_borrow">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>借阅</string>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_password"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
